src/sentry/static/sentry/app/components/charts/baseChart.jsx

```diff
@@ -0,0 +1,98 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactEchartsCore from 'echarts-for-react/lib/core';
+import 'zrender/lib/svg/svg';
```

Comment:
what's this for? how's it used?

Reply:
This is required for the svg renderer to work.